CHANGELOG.md

```diff
@@ -10,6 +10,7 @@
 
 - Reduce the size of docker image ([#272](https://github.com/src-d/sourced-ui/issues/272))
 - Make the current source{d} palettes accessible for color blindness impairments ([#302](https://github.com/src-d/sourced-ui/issues/302))
+- Convert the current `srcdMain` and `srcdAll` palettes into interlaced ones, and rename the old gradient ones as `srcdMainGradient` and `srcdAllGradient` ([#305](https://github.com/src-d/sourced-ui/pull/305))
```

Comment:
Could you move this both to `Changed`? They're not actually internal changes, as they have an impact on the end-user.

Reply:
That's totally true. I swear I believed I was adding it into `Changed` section.
99d12d3c69b5e437d9d7b5e1e2f54b595f529866